source/source/PlayState.hx

```diff
@@ -23,6 +25,7 @@ class PlayState extends FlxState
 	{
 		super.create();
         container.addDefaultSystems(this);
+        container.addSystem(new MouseShootSystem(this));

```

Comment:
I don't think this should be part of Nebula. I think it should be part of Planetoid. (As a consequence of this, we need a way to add non-default systems to Nebula's chain of systems.)